src/_shared/svgs/camera.svg

```diff
@@ -0,0 +1 @@
+<svg width="18" height="13" viewBox="0 0 18 13"><path d="M18 3.5v8l-1.5 1.5h-15l-1.5-1.5v-8l1.5-1.5h3.5l2-2h4l2 2h3.5l1.5 1.5zm-9 7.5c1.9 0 3.5-1.6 3.5-3.5s-1.6-3.5-3.5-3.5-3.5 1.6-3.5 3.5 1.6 3.5 3.5 3.5z"/></svg>
```

Comment:
The SVG files should be present on master following this merge: https://github.com/guardian/commercial-templates/commit/ff0b235fc48c6d8aee025331dab74c843ed0ad0c